asset-registry/src/lib.rs

```diff
@@ -0,0 +1,242 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use frame_support::pallet_prelude::*;
+use frame_support::traits::EnsureOrigin;
+use frame_support::transactional;
```

Comment:
use tight format: `use frame_support::{....}`, also other one in this file and other files needs to tidy-up.